tests/aie_runtime/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 The IREE Authors
```

Comment:
can we move these in the runtime directory (which i suggested be renamed in the earlier comment).